model/riscv_csr_map.sail

```diff
@@ -113,6 +172,8 @@ mapping clause csr_name_map = 0x143  <-> "stval"
 mapping clause csr_name_map = 0x144  <-> "sip"
 /* supervisor protection and translation */
 mapping clause csr_name_map = 0x180  <-> "satp"
+/* supervisor envcfg */
+mapping clause csr_name_map = 0x10A  <-> "senvcfg"
```

Comment:
This is also included in #395, maybe makes sense to submit this as a tiny bugfix PR that can hopefully be merged in a reasonable timeframe.

Reply:
Good idea, I'll do that.

Reply:
In #405 ; I'll remove it from this.

Reply:
Done. I've also added `haveZihpm()`.